.github/labeler.yml

```diff
@@ -1 +1,29 @@
-'Type: RFC': 'rfcs/**/*.md'
+'Type: RFC':
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the FluentUIV0 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33202&commitId=a33c1b08904211d5f6633c08be4273aa93daf8e3&env=prod&clientType=FLUENTUI)


  <details><summary><b>Divider</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Divider.DividerVerticalExampleShorthand.chromium.png|319| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v0"/>